x/incentive/keeper/keeper_fees.go

```diff
@@ -2,28 +2,89 @@ package keeper
 
 import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	ammtypes "github.com/elys-network/elys/x/amm/types"
 	"github.com/elys-network/elys/x/incentive/types"
+	ptypes "github.com/elys-network/elys/x/parameter/types"
 )
 
 // Move gas fees collected to incentive module
-// TODO:
-// Convert Elys to USDC
 func (k Keeper) CollectGasFeesToIncentiveModule(ctx sdk.Context) sdk.Coins {
 	// fetch and clear the collected fees for distribution, since this is
 	// called in BeginBlock, collected fees will be from the previous block
 	// (and distributed to the previous proposer)
 	feeCollector := k.authKeeper.GetModuleAccount(ctx, k.feeCollectorName)
 	feesCollectedInt := k.bankKeeper.GetAllBalances(ctx, feeCollector.GetAddress())
 
-	// transfer collected fees to the distribution module account
-	err := k.bankKeeper.SendCoinsFromModuleToModule(ctx, k.feeCollectorName, types.ModuleName, feesCollectedInt)
+	// Get pool Ids that can convert Elys
+	poolIds := k.amm.GetAllPoolIdsWithDenom(ctx, ptypes.Elys)
```

Comment:
Last time, we have done a communication to have a pool id to be configured on params to reduce complexity.
Can you apply that?

Reply:
@jelysn 
I assumed there will be several pools that contains the same denom. For example, we may have elys:usdc, juno:usdc, elys:juno:usdc etc. If I put elys as denom, then I will have 2 pool ids. So I can't put pool id there. Find out all possible pool ids and then go from there.

Reply:
@jelysn 
Even in that case, denom is the best param to get a pool id. There is no way to get pool id without looping all pools and compare its assets denom. Any better idea?